recipes/insilicoseq/run_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+curl -L https://ndownloader.figshare.com/files/15127220 > bogus-genome.fasta
+curl -L https://ndownloader.figshare.com/files/15127223 > checksums.sha
```

Comment:
How big are those? Can we have them in-repo?

@bioconda/core - I think we need a directory in the repo, or a second repo, with some stanard dest-data that tools can share. This repo is a beast already, adding more and more testdata will not help, but without many things cannot be verified properly.

Reply:
@epruesse That was my strategy with https://github.com/bioconda/bioconda-recipes/pull/15240/commits/3aad15ed63bcf5d7439a2becc795aa0f8d2e31eb. Alas, the file could not be found.

```
FileNotFoundError: [Errno 2] No such file or directory: 'bogus-genome.fasta'
```

See https://circleci.com/gh/bioconda/bioconda-recipes/56296.

Reply:
See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-section

You can have the test files as part of the source, then they need to be listed as `source_files` under `test`, or as files in the recipe folder, then they need to be listed as `files` under `test`. That should work - we've got plenty recipes. Might want to zip them, they won't change so no need to keep them clear text. The smaller the better.

Reply:
Catch with the figshare hosting is that the files might go away.

Reply:
Opened an issue #536

Reply:
figshare is actually a fairly stable archival service, but yes I agree it's not ideal. I'll see if I can dig up an example with test files in the recipe.

Reply:
Ok, test files are bundled *in situ*.